python/requirements.txt

```diff
@@ -1,10 +1,4 @@
-Flask>=0.12.3
-Jinja2>=2.9.5
-MarkupSafe==0.23
-Werkzeug>=0.14
-click==6.7
-itsdangerous==0.24
-requests==2.20.0
-wsgiref==0.1.2 ; python_version < '3.2'
-plaid-python>=5.0.0, < 6.0.0
-python-dotenv==0.14.0
+Flask==1.1.2
+plaid_python==8.0.0b9
+# plaid==0.1.7
```

Comment:
```suggestion
```

Also where did the other dependencies go? should we freeze them via this requirements.txt file? I dont know how to do that with pip but i assume there is a way

Reply:
so I put the other dependencies back in and it broke it, so took em back out.  I vaguely recall you and I changing this requirements file and specifically not putting those dependencies in.